RocketChat/Rocket.Chat.ReactNative
# feat: mobile color normalization

## Proposed changes

Changes the color tags to use the standard that is currently used on the web, normalizing the mobile app's themes and color usage.

## Issue(s)	


## How to test or reproduce


- From within the app, change the themes, test all possible combinations and see if you notice any discrepancies in the colors.

## Screenshots

## Types of changes



- [x] Improvement - BREAKING CHANGE

## Checklist



## Further comments

Jira: [NATIVE-506]

[NATIVE-506]: https://rocketchat.atlassian.net/browse/NATIVE-506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
